core/jazz_usermanagement/config/dev-config.json

```diff
@@ -1,23 +1,29 @@
 {
-  "USER_POOL_ID":"{user_pool_id}",
+  "USER_POOL_ID": "{user_pool_id}",
   "REGION": "{region}",
   "USER_CLIENT_ID": "{user_client_id}",
-  "reg_codes": ["JAZZ"],
-  "required_fields": ["userid", "userpassword", "usercode"],
+  "reg_codes": [
+    "JAZZ"
+  ],
+  "required_fields": [
+    "userid",
+    "userpassword",
+    "usercode"
+  ],
   "SCM_TYPE": "{scm_type}",
-  "SCM_CONFIG" : {
-    "bitbucket" : {
-      "HOSTNAME": "{bb_service_host}",
+  "SCM_CONFIG": {
+    "bitbucket": {
+      "BASE_URL": "{base-url}",
```

Comment:
maybe call the replacement key as `scm_base_url`

Reply:
The key name was correct earlier as the value is a HOSTNAME and not a URL

Reply:
Done